utils/build.js

```diff
@@ -1,5 +1,5 @@
-const mode = process.argv.indexOf('production') != -1 ? 'production' : 'development';
-process.env.NODE_ENV = process.env.NODE_ENV || mode;
+// const mode = (process.argv.indexOf('production') || process.argv.indexOf('android')) != -1 ? 'production' : 'development';
+// process.env.NODE_ENV = process.env.NODE_ENV || mode;
```

Comment:
Uncomment this. When this is commented out, it will never build in production mode.

Edit: seems like you may be able to get it to build production via setting env variables, but cli args are preferred.